test/models/ability_test.rb

```diff
@@ -713,7 +713,7 @@ def teardown
     team_perms = [
       "bulk_create Tag", "bulk_update ProjectMedia", "create TagText", "read Team", "update Team", "destroy Team", "empty Trash",
       "create Project", "create Account", "create TeamUser", "create User", "create ProjectMedia", "invite Members",
-      "not_spam ProjectMedia", "restore ProjectMedia", "confirm ProjectMedia", "update ProjectMedia", "duplicate Team",
+      "not_spam ProjectMedia", "restore ProjectMedia", "confirm ProjectMedia", "update ProjectMedia", "duplicate Team", "create Feed",
```

Comment:
Plz lmk when this is merged so I update the frontend with the right permission check

Reply:
Will do!

Reply:
@amoedoamorim this is merged now.